apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2.java

```diff
@@ -868,10 +893,40 @@ private void addC4DicPayor(Coverage coverage) {
             coverage, TransformerUtilsV2.PROVIDER_ORG_ID, Profile.C4DIC);
     organization.setActive(true);
     organization.setName(COVERAGE_ISSUER);
+    addC4dicOrganizationContact(organization);
 
     coverage.addPayor(new Reference(organization));
   }
 
+  /**
+   * Sets the Payor's contact information on the contained {@link Organization} resource.
+   *
+   * @param organization The contained {@link Organization} resource
+   */
+  private void addC4dicOrganizationContact(Organization organization) {
+    Organization.OrganizationContactComponent organizationContactComponent =
+        new Organization.OrganizationContactComponent();
+    organizationContactComponent.setPurpose(
+        TransformerUtilsV2.createCodeableConcept(
+            TransformerConstants.C4DIC_CONTACT_TYPE_CODE_SYSTEM,
+            null,
+            TransformerConstants.C4DIC_CONTACT_TYPE_PAYOR_DISPLAY,
+            TransformerConstants.C4DIC_CONTACT_TYPE_PAYOR_CODE));
+
+    ContactPoint phoneContact =
+        TransformerUtilsV2.createContactPoint(
+            ContactPoint.ContactPointSystem.PHONE,
+            TransformerConstants.C4DIC_MEDICARE_SERVICE_PHONE_NUMBER,
+            null);
+
+    ContactPoint emailContact =
+        TransformerUtilsV2.createContactPoint(
+            ContactPoint.ContactPointSystem.EMAIL, TransformerConstants.C4DIC_MEDICARE_EMAIL, null);
```

Comment:
As far as I can tell, there isn't actually a publicly listed email address to contact on medicare.gov. Maybe it's better to use `ContactPointSystem.URL` here instead?

Reply:
You're right! Good catch!